docs/static/azure-module.asciidoc

```diff
@@ -186,27 +157,67 @@ global configuration is effectively ignored.
 -----
 modules:
   - name: azure
-    var.elasticsearch.hosts: "localhost:9200"
-    var.kibana.host: "localhost:5601"
-    var.input.azure_event_hubs.threads: 8
-    var.input.azure_event_hubs.decorate_events: true
-    var.input.azure_event_hubs.consumer_group: logstash
+    var.elasticsearch.hosts: localhost:9200
+    var.kibana.host: localhost:5601
+    var.input.azure_event_hubs.decorate_events: true <1>
+    var.input.azure_event_hubs.threads: 9
+    var.input.azure_event_hubs.consumer_group: "logstash"
+    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=instance1..."
     var.input.azure_event_hubs.event_hubs:
-      - ["name",                      "event_hub_connection",      "storage_connection",                                      "initial_position", "decorate_events"]
-      - ["insights-operational-logs", "Endpoint=sb://example1...", "DefaultEndpointsProtocol=https;AccountName=example1....", "HEAD",             "true"]
-      - ["insights-metrics-pt1m",     "Endpoint=sb://example2...", "DefaultEndpointsProtocol=https;AccountName=example2....", "TAIL",             "true"]
-      - ["insights-logs-errors",      "Endpoint=sb://example3...", "DefaultEndpointsProtocol=https;AccountName=example3....", "TAIL",             "false"]
-      - ["insights-operational-logs", "Endpoint=sb://example4...", "DefaultEndpointsProtocol=https;AccountName=example4....", "HEAD",             "true"]
+      - ["name",                                    "initial_position",  "storage_container",  "decorate_events",  "event_hub_connection"]                                   <2>
+      - ["insights-operational-logs",                 "TAIL",              "activity-logs1",    "true",             "DefaultEndpointsProtocol=https;AccountName=instance1"]
+      - ["insights-operational-logs",                 "HEAD",              "activity_logs2",<3>   "true",             "DefaultEndpointsProtocol=https;AccountName=instance2"] 
```

Comment:
nitpick: i like changing one of these from TAIL to HEAD ... but can you change a different one to HEAD other then insights-operational-logs